core/design/src/main/kotlin/edu/stanford/spezi/core/design/theme/Elevations.kt

```diff
@@ -0,0 +1,25 @@
+package edu.stanford.spezi.core.design.theme
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.ReadOnlyComposable
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+
+object Elevations {
+    object Card {
+        val small: Dp
+            @Composable
+            @ReadOnlyComposable
+            get() = 2.dp
+
+        val medium: Dp
+            @Composable
+            @ReadOnlyComposable
+            get() = 4.dp
+
+        val large: Dp
+            @Composable
+            @ReadOnlyComposable
+            get() = 8.dp
+    }
+}
```

Comment:
i thought it makes sense to have it standardised with small medium and large, so that if someone creates a new module they can be sure that they have access to all the other common sizes and also if someone external creates an app with the framework then they don't have to start defining their own sizes, but then i delete it for now